client/src/main/java/io/pravega/client/admin/StreamManager.java

```diff
@@ -183,10 +184,25 @@ public static StreamManager create(ClientConfig clientConfig) {
      * @param scopeName  The name of the scope to delete.
      * @param forceDelete To list and delete streams, key-value tables and reader groups in scope before attempting to delete scope.
      * @return True if scope is deleted, false otherwise. 
-     * @throws DeleteScopeFailedException is thrown if this method is unable to seal and delete a stream.  
+     * @throws DeleteScopeFailedException is thrown if this method is unable to seal and delete a stream.
+     *
+     * @deprecated As of Pravega release 0.11.0, replaced by {@link #deleteScopeRecursive(String)}.
      */
+    @Deprecated
     boolean deleteScope(String scopeName, boolean forceDelete) throws DeleteScopeFailedException;
 
+    /**
+     * Deletes scope by listing and deleting all streams/RGs/KVTs in scope.
+     * This method is atomic and new streams/RGs/KVTs can not be added/updated if this
+     * method is called. Deleting scope is idempotent and failure to
```

Comment:
All Controller side operations are idempotent and get retried. I do not see the need to mention this.

Reply:
updated as per suggestion